geode-core/src/main/java/org/apache/geode/internal/admin/ClientHealthMonitoringRegion.java

```diff
@@ -76,7 +80,9 @@ private static void initialize(InternalCache cache) {
       factory.setScope(Scope.LOCAL);
       factory.setEntryTimeToLive(
           new ExpirationAttributes(ADMIN_REGION_EXPIRY_INTERVAL, ExpirationAction.DESTROY));
-      cache.getLogger().fine("ClientHealthMonitoringRegion, setting TTL for entry....");
+      if (logger.isDebugEnabled()) {
+        logger.debug("ClientHealthMonitoringRegion, setting TTL for entry....");
```

Comment:
Do we really need to check `isDebugEnabled` when the `logger.debug` will do that for us too and there is no "work" being done here?

Reply:
Yes, we need to check isDebugEnabled. This hits a single volatile boolean instead of what logger.debug does (it's a lot more).

Reply:
There was a FastLoggerBenchmark that I added to Geode this year, but it's now missing. I'm not sure who or how it was deleted, but that Benchmark has the answers to your questions.

Reply:
I still can't find the benchmark, but it was last year (not this year).

Reply:
I found it hiding in an unmerged branch (no comment). I'll file a PR and tag you.